en/guide/message_schema.md

```diff
@@ -0,0 +1,248 @@
+* `value` (optional): The *value* for the entry (a number).
+* `description` (optional): A description of the entry.
+* [param](#param) (optional): A param value used in a [MAV_CMD](../messages/common.md#MAV_CMD) enum entry.
+* [deprecated](#deprecated) / [wip](#wip) (optional): A tag indicating that the enum is deprecated or "work in progress". 
+
+#### param {#param}
+
+The `<param>` tag is used in the [MAV_CMD](../messages/common.md#MAV_CMD) enum as part of defining mission commands.
+Params include an attribute `index` (with value from 1 to 7) and contain a parameter description string.
+
+For example, see [MAV_CMD_NAV_PAYLOAD_PLACE](../messages/common.md#MAV_CMD_NAV_PAYLOAD_PLACE)
+
+```xml
+<entry value="94" name="MAV_CMD_NAV_PAYLOAD_PLACE">
+    <description>Descend and place payload. Vehicle descends until it detects a hanging payload has reached the ground, the gripper is opened to release the payload</description>
+    <param index="1">Maximum distance to descend (meters)</param>
+    <param index="2">Empty</param>
+    <param index="3">Empty</param>
+    <param index="4">Empty</param>
+    <param index="5">Latitude (deg * 1E7)</param>
+    <param index="6">Longitude (deg * 1E7)</param>
+    <param index="7">Altitude (meters)</param>
+</entry>
+```
+
+> **Note** Mission commands are encoded in [MISSION_ITEM](../messages/common.md#MISSION_ITEM) or [MISSION_ITEM_INT](../messages/common.md#MISSION_ITEM_INT) messages as part of the [Mission Protocol](../protocol/mission.md). 
+  Some commands can be sent outside of missions in [COMMAND_INT](../messages/common.md#COMMAND_INT) or [COMMAND_LONG](../messages/common.md#COMMAND_LONG) messages.
+
+
+
+
+### Others Tags
+
+This section lists a number of tags can be used in a number of other types - e.g. messages and enums.
+
+#### deprecated {#deprecated}
+
+The `<deprecated>` tag can be used in an [enum](#enum), enum [entry](#entry) (value) or [message](#message) to indicate that the item has been superseded. The tag also indicates the time of deprecation and the replacement item.
+
+The generator toolchain can be configured to conditionally build messages omitting the `deprecated` entries.
+
+> **Tip** An entity should be marked as deprecated only when the main users have had an opportunity to update to the new method.
+
+As a concrete example, below we see that [SET_MODE](../messages/common.md#SET_MODE) is deprecated and replaced by [MAV_CMD_DO_SET_MODE](../messages/common.md#MAV_CMD_DO_SET_MODE) on `2015-12`.
+```xml
+    <message id="11" name="SET_MODE">
+      <deprecated since="2015-12" replaced_by="MAV_CMD_DO_SET_MODE">Use COMMAND_LONG with MAV_CMD_DO_SET_MODE instead</deprecated>
+```
+
+The `deprecated` attributes are:
+* `since`: Year/month when deprecation started. Format: YYYY-MM.
+* `replaced by`: String of entity that supersedes this item.
+* `description` (optional): String with more information about the deprecation.
```

Comment:
description is not an attribute, but an element, hence it is inside the <deprecated>  xxx <deprecated/>